client/src/components/ReduxTextField/index.js

```diff
@@ -6,6 +6,7 @@ const ReduxTextField = (props) => {
 
   return (
     <TextField
+    style={{width:"60%", paddingBottom: "1.5rem"}}
```

Comment:
@meganhng I don't think we should include these style properties here. Any styling that we want to apply to `ReduxTextField` should be passed as a style prop. That way, we ensure that this component is more reusable. For example, we're using `ReduxTextField` for signing in and signing up, but we also use this component when we input the title and description for a new task. The styling you add here might not be ideal in that use case, so it would be better to pass `{ width:"60%", paddingBottom: "1.5rem" }` as a prop from the signin and signup pages.

Reply:
fixed 😄